packages/moleculer-db-adapter-sequelize/src/index.js

```diff
@@ -52,33 +52,35 @@ class SequelizeDbAdapter {
 	 */
 	connect() {
 		const sequelizeInstance = this.opts[0];
+		let noSync = false;
 
-		if (sequelizeInstance && sequelizeInstance instanceof Sequelize)
+		if (sequelizeInstance && sequelizeInstance instanceof Sequelize) {
 			this.db = sequelizeInstance;
-		else
+			if (this.opts[1] === "noSync") {
+				noSync = true;
```

Comment:
Why need these use-case?

Reply:
I add it because you already have the option to noSync the model and in case you pass the Sequelize instance to the module, there is no way to disable the model Sync. There is only an if for SQLite db